pyproject.toml

```diff
@@ -35,7 +35,7 @@ qualang-tools = { version = "==0.14.0", optional = true}
 setuptools = { version = ">67.0.0", optional = true }
 laboneq = { version ="==2.7.0", optional = true }
 # TODO change qibosoq when release
-qibosoq = { git = "https://github.com/qiboteam/qibosoq.git", branch = "main", optional = true}
+qibosoq = { git = "https://github.com/qiboteam/qibosoq.git", branch = "split_programs", optional = true}
```

Comment:
Can you put back `main` now?
Or even a release? (if possible to add one in Qibosoq)

Reply:
Yes I think now it should be ready for a release!

Reply:
https://github.com/qiboteam/qibosoq/issues/44